BEST-PRACTICES.md

```diff
@@ -17,15 +17,24 @@ We recommend the following coding practices for high quality contributions:
 - Prefer self-explanatory code as much as possible, but provide helpful comments for complex expressions and code blocks.
 - Add unit tests for any new or changed functionality, and update any existing tests that are impacted by your changes.
   - SUSHI uses [Jest](https://jestjs.io/) as a testing framework.
-  - To run the full test suite, run `npm test`. Ensure all tests are passing.
+  - Ensure all tests are passing. Ensure that code coverage of the new code is complete.
+  - To run the full test suite, run `npm test`.
+  - To review the test coverage report, run `npm run coverage` after running the full test suite.
 - Follow the code style and conventions as enforced by the lint configuration and as evidenced by the existing code.
   - SUSHI uses [ESLint](https://eslint.org/) for code linting.
-  - To run the linter on all code, run `npm run lint`. Ensure there are no issues reported.
+  - Ensure there are no issues reported.
+  - To run the linter on all code, run `npm run lint`.
+  - To automatically fix as many issues as possible, run `npm run lint:fix`. This uses ESLint's [--fix](https://eslint.org/docs/latest/use/command-line-interface#fix-problems) option.
```

Comment:
I think these last two bullets (To run..., To automatically fix...) should be nested below "Ensure...". Or change the order around a bit.

Reply:
Changed the order in [7c4b6fe](https://github.com/FHIR/sushi/pull/1474/commits/7c4b6fee2205e44567d370c0c1b861c3ba6a4746)